src/object.d

```diff
@@ -1737,7 +1728,7 @@ class Throwable : Object
                     sink("\n"); sink(t);
                 }
             }
-            catch (Throwable)
+            catch (Exception)
             {
```

Comment:
`Exception` implies that the program is still in a recoverable state. `Error`s imply some unrecoverable state.